src/main/java/com/elytradev/smores/block/BlockOre.java

```diff
@@ -64,4 +63,12 @@ public int getHarvestLevel(IBlockState state) {
 	public String getHarvestTool(IBlockState state) {
 		return "pickaxe";
 	}
+
```

Comment:
I've never liked the oredict being in the blocks - that's a nitpick, not a *must correct* - to me it's like recipes where it's definitely about the conventions around a block, but not really about the block itself. Oredict assignments ideally go after the block registrations (so that they happen before recipes), and you have all this information available then too.

Reply:
It's just a force of habit by now, figuring a uniform interface for Oredict registration just makes it easier to forget about the registration as a whole.